### DISCUSSION TITLE
Add Amazon Glue as a producer?

### DISCUSSION THREAD

**Discussion by suraj5077** [OpenLineage/OpenLineage#2378] [General]
Is Amazon Glue a supported OL producer? In the docs, there is no mention. 
If not, is there any plan for building this integration? 
And if it is already supported can we add some docs on how to configure and test it?

**Comment by Sheeri** [comment]
There is not a specific Amazon Glue integration, but there is a spark integration that can be used with AWS Glue - https://openlineage.io/docs/integrations/spark/
